### PR TITLE
chore: cleanup GPT properties

### PR DESCRIPTION
**Pull request by schultzp2020**
## Which issue(s) does this PR fix

Fixes https://github.com/janus-idp/software-templates/issues/143

### CODE REVIEW

**Review by PatAKnight — APPROVED** [review]
LGTM

File: scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml [hunk 1]
```diff
@@ -18,7 +18,11 @@ metadata:
     {%- endif %}
     backstage.io/kubernetes-id: ${{ values.repoName }}
     backstage.io/techdocs-ref: dir:.
-    github.com/project-slug: ${{ values.githubOrg }}/${{ values.repoName }}
+    {%- if values.sourceControl == "gitlab.com" %}
```
Comment (kadel) [inline scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml:21]: Would it make sense to do it more generic? Something like 
 
```yaml
{%- if values.sourceControl %}
    ${{values.sourceControl}}/project-slug: ${{ values.orgName }}/${{ values.repoName }}
{%- endif %}
```

Comment (schultzp2020) [reply scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml:21]: Ideally that would be nice; however, `${{values.sourceControl}}/project-slug:` is not standardized for the source control plugins. e.g. [Azure DevOps](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md#entity-annotation)

Comment (kadel) [reply scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml:21]: uff, ok. This if-else block is going to look "interesting" as we add more and more :-/ , guess not much we can do about it than.

**Review by kadel — APPROVED** [review]

**Pull request merged**